src/extension/app/store/site.js

```diff
@@ -206,7 +206,13 @@ export class SiteStore {
     if (owner && repo) {
       // look for custom config in project
       try {
-        const res = await callAdmin({ owner, repo, ref }, 'sidekick', '/config.json');
+        const res = await callAdmin(
+          {
+            owner, repo, ref, adminVersion,
```

Comment:
Unrelated to this PR: noticed during testing that a CI admin version is not being applied to this admin call.